MichaelEWalker87/hang-in-there-boilerplate
# Create Random Poster Feature & Create Show/Hide Feature

##  Pull Request Template

### What’s this PR do? 
- Loads random poster image, title, and quote on page load or refresh.
- Loads random poster image, title, and quote at button click.
- Shows "Make Your Own Poster" form on button click and hides rest of page.
- Shows "Saved Poster" page on button click and hides rest of page.
- Redirects back to home of page from "Make Your Own Poster" form on button click.
- Redirects back to home of page from "Saved Poster" page on button click

### Where should the reviewer start?

`main.js`

### How should this be manually tested?

- Load the page and refresh it multiple times to see if the home poster is changing.
- Click the [Show Another Random Poster] button multiple times to see if the home poster is changing.
- Click the [Make Your Own Poster] button to see the form.
- Click the [Nevermind, Take Me Back] button to return to home.
- Click the [Show Saved Posters] button to see the saved posters section.
- Click the [Back To Main] button to return to home.

### Any background context you want to provide?

We did not know the norms for frequency and naming conventions of branches, commits and pull requests. They will be more consistent in all branches after this branch.

### What are the relevant tickets? 

- If you go to the Form and then saved posters, the Form shows up again instead of saved posters.
- If you go to Saved Posters then back to home and then back to Saved Posters, the button [Back To Main] is missing.
